### PR TITLE
fix bug - The error message should be prompted after send SMS and u i problem

### PR DESCRIPTION
**Pull request by caffffe**

### CODE REVIEW

File: src/components/ComposeTextPanel/index.js [hunk 1]
```diff
@@ -215,12 +215,14 @@ ComposeTextPanel.propTypes = {
...
     name: PropTypes.string.isRequired,
   })).isRequired,
+  internalSMS: PropTypes.bool,
 };
 
 ComposeTextPanel.defaultProps = {
   messageText: '',
   typingToNumber: '',
   senderNumber: '',
+  internalSMS: '',
```
Comment (embbnux) [inline src/components/ComposeTextPanel/index.js]: default true ?

File: src/components/ComposeTextPanel/index.js [hunk 2]
```diff
@@ -40,7 +40,7 @@ class ComposeTextPanel extends Component {
     super(props);
 
     this.state = {
-      showAlert: !this.hasSenderNumbers(),
+      showAlert: !this.hasSenderNumbers() && !this.props.internalSMS,
```
Comment (embbnux) [inline src/components/ComposeTextPanel/index.js]: showAlert when user has sms permission and have no SenderNumbers

**Comment by u9520107** [comment]
@caffffe Remember to branch off from master when you want to fix specific bugs, rather than merging master to whatever branch you're working on.

File: src/components/MessageItem/styles.scss [hunk 3]
```diff
@@ -72,6 +72,7 @@
   text-overflow: ellipsis;
   overflow: hidden;
   max-width: 20em;
+  margin-right:2em;
```
Comment (embbnux) [inline src/components/MessageItem/styles.scss]: space

**Pull request merged**